public/app/features/explore/QueryLibrary/QueryTemplateForm.tsx

```diff
@@ -61,7 +63,11 @@ export const QueryTemplateForm = ({ onCancel, onSave, queryToAdd, templateData }
     queryToAdd !== undefined ? [queryToAdd] : templateData?.query !== undefined ? [templateData?.query] : [];
 
   const handleAddQueryTemplate = async (addQueryTemplateCommand: AddQueryTemplateCommand) => {
-    return addQueryTemplate(addQueryTemplateCommand)
+    return addQueryTemplate({
+      namespace: getK8sNamespace(),
+      comGithubGrafanaGrafanaPkgApisPeakqV0Alpha1QueryTemplate:
```

Comment:
this... is an ugly name 😅 is this something we have control over in the api spec?

Reply:
I believe the source of this name is something like the fully resolved module + type name in go, i presume it's something like `com.github.grafana/grafana/grafana/pkg/apis/peakq/v0/alpha1.QueryTemplate`, and whatever generates the API Spec in the backend doesn't put too much effort in making nice names.